### PR TITLE
add nvmrc file, default node version 16.17.0

### PR DESCRIPTION
**Pull request by alejandroaguilarhiguera**
You can do `nvm use`

### CODE REVIEW

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
Looks good! will approve after pre-commit passes.

**Review by notmd — APPROVED** [review]

**Comment by notmd** [comment]
Hmm some how I can't merge this

**Review by fozziethebeat — APPROVED** [review]

**Comment by fozziethebeat** [comment]
Seems that my request change review blocks others from merging.  I'll have to think more carefully on when to review with that setting

**Pull request merged**
